content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -601,6 +601,12 @@ parameters:
 
 ### Quobyte
 
+{{< feature-state for_k8s_version="v1.22" state="deprecated" >}}
+
+The Quobyte storage plugin is deprecated, an 
```

Comment:
ugh... maybe this should say, an example for the new out-of-tree plugin or something like that

Reply:
I agree, differentiate the in-tree example with the link to the out-of-tree example